crates/serpen/tests/fixtures/comprehensive_ast_rewrite/main.py

```
@@ -0,0 +1,103 @@
+
+    # Variable name reuse and conflicts
+    result = db_result + util_result + auth_result  # Shadows global result
+
+    # Using relative imports
+    base_result = base.initialize()
+
+    # Complex expression with multiple conflicts
+    final_result = {
+        "process_results": [db_result, util_result, auth_result],
+        "validation_results": [local_validate_result, auth_validate_result, util_validate_result],
+        "user_types": [auth_user.name, model_user.name, service_user.username],
```

Pylint (3.3.7)

[error] Instance of 'User' has no 'username' member

(E1101)